packages/dhis2/src/Adaptor.js

```diff
@@ -35,6 +35,12 @@ export function execute(...operations) {
   };
 
   return state => {
+    const version = state.configuration?.apiVersion;
+    if (+version >= 42) {
+      console.warn(
+        `This adaptor is incompatible with DHIS2 API verison 42. (see here: https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/tracker.html).`
```

Comment:
I would remove the brackets - otherwise this is good.

Oh, let's add WARNING in all caps at the start. I know  you're logging to console.warn but that's often not very clear in the output (especially in lightning)